### ISSUE TITLE
SickRage :: Unable to determine free space

### ISSUE THREAD

**Issue by heymoe** [SickRage/SickRage#1309]
**Branch/Commit:** master / 89520a34e67073998be844c3721a3730341abb12
**OS:** CentOS v5.11 using Python 2.7.10

**What you did:** Updated Sickrage? I can run "df" as both root and the user SickRage is running as and get free space information from it. I'm not seeing any errors or warnings in the system / OS logs that might be causing this issue.

**What happened:** I noticed that the "Service Status" page is no longer displaying free space for the different paths I have configured under the "Disk Space" section. It's now reporting "Missing". I'm also seeing the following warnings in the logs:

- Thread-13 :: Unable to determine free space
- Thread-13 :: Unable to determine free space, so I will assume there is enough.

**What you expected:** Expected SickRage to be able to determine free space. This seems to have started around 2016-03-29 so I'm guessing after I did an update.

**Logs:** I did enable Debug logging but it doesn't seem to catch any debug information for this issue.

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by heymoe** [comment]
I know this is closed but just wanted to confirm upgrading to:

master / f59e46744829adf67a73a7dc9442eda26c40301d

which contains the fixes does indeed fix this problem.. Thanks!

**Comment by miigotu** [comment]
Thank you for confirming =P

**Comment by FGOD1983** [comment]
I know this is an old closed issue, but seems I am running in the same problem after replacing my disk with a new one that has the same folder structure as the old one and has the old content copied to it.

Sickchill still reports the free space of my 6TB (which was also replaced last month, but my 8TB NAS drive that replaced my old 4TB drive is reported as missing in the free space part of server status. 

This happened after turning of my server to replace the disk and then turn my server on again.

I also tried to restart sickchill but that had no effect on this.

important info you might need:


SickChill Info: Branch: master
                       Commit: 1ed315660d99f6673a3bb53363fdff6902b89429
                       Version: v2019.06.28-1
                       Database Version: 44.0

Python Version: 2.7.12 (default, Nov 12 2018, 14:36:49) [GCC 5.4.0 20160609]

SSL Version: OpenSSL 1.0.2g 1 Mar 2016

OS: Linux-4.4.0-159-generic-x86_64-with-Ubuntu-16.04-xenial
